app/_locales/en/messages.json

```diff
@@ -1832,6 +1838,9 @@
   "newTransactionFee": {
     "message": "New Transaction Fee"
   },
+  "newValues": {
+    "message": "new values"
+  },
```

Comment:
May be `advancedGasFeeNewValues `

Reply:
If for some scenario for resuing `new values` occur in the future, won't it be better for the copy to have a neutral name?